src/snowflake/cli/_plugins/connection/util.py

```diff
@@ -104,9 +59,9 @@ def is_regionless_redirect(conn: SnowflakeConnection) -> bool:
     """
     try:
         return (
-            get_ui_parameter(
-                conn, UIParameter.NA_ENABLE_REGIONLESS_REDIRECT, "true"
-            ).lower()
+            get_snowflake_facade()
+            .get_ui_parameter(UIParameter.NA_ENABLE_REGIONLESS_REDIRECT, "true")
```

Comment:
I agree with your point about the current facade staying NA specific. And I think there will likely be more cases where we have "general" or "shared" methods between NA and other plugins/domains.
So when it comes to where to put `get_ui_parameter` it might be worth making a decision on the shared/general methods here?
One suggestion could be to have them in the `BaseSQLExecutor` which the Facade(s) inherit from. what do you think?

Reply:
That's what I originally suggested, IIRC, but IMHO not worth doing in this particular PR. I don't think we should replace the general utils either at this time. @sfc-gh-melnacouzi please revert this change?